apps/ios/Shared/Views/Call/ActiveCallView.swift

```diff
@@ -117,7 +135,9 @@ struct ActiveCallView: View {
                 call.callState = .connected
                 call.connectionInfo = connectionInfo
             case .ended:
-                closeCallView(webView)
+                DispatchQueue.main.async {
```

Comment:
The difference between this and `await MainActor.run` is that the latter would wait for the completion of the code inside - in most cases it is what you want, there are only rare cases when you want concurrent execution of code inside and after.